panoptes_client/subject_set.py

```diff
@@ -34,4 +34,15 @@ def add_subjects(self, subjects):
             json={'subjects': _subjects}
         )
 
+    def add_subject_link(self, subject_id):
+        self.post(
+            '{}/links/subjects'.format(self.id),
+            json={'subjects': [subject_id]}
+        )
+
+    def delete_subject_link(self, subject_id):
```

Comment:
I'd rename this to `remove_subjects` and make it work like `add_subjects`, so it could take a list or a single `Subject`